src/components/course/CourseParticipants.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useContext, useState } from 'react';
+import { Col, Empty, List, Row } from 'antd';
+import SearchParticipant from './SearchParticipant';
+import SortParticipant from './SortParticipant';
+import AccentColorButton from '../button/AccentColorButton';
+import IssueCertificateModal from '../Modals/IssueCertificateModal';
+import classes from '../../routes/CourseDetail.module.scss';
+import { ModalContext } from '../../context/ModalContext';
+import { CourseParticipant, ParsedCourseOverview } from '../../types/Course';
+
+interface Props {
+  course: ParsedCourseOverview;
+  participantList: CourseParticipant[];
+  setEnteredFilter: (filter: string) => void;
+  setParticipantList: (participantList: CourseParticipant[]) => void;
+}
+
+export default function CourseParticipants(props: Props) {
+  const modalContext = useContext(ModalContext);
+
+  const [isSelecting, setSelecting] = useState(false);
+  const [selectedParticipants, setSelectedParticipants] = useState<
+    CourseParticipant[]
+  >([]);
+
+  if (!props.course.subcourse) {
+    return null;
+  }
+
+  if (props.course.subcourse.participants === 0) {
+    return <Empty description="Du hast noch keine Teilnehmer*innen" />;
+  }
+
+  const resetSelect = () => {
+    setSelectedParticipants([]);
+    setSelecting(false);
+  };
+
+  return (
+    <div>
+      <div>
+        <Row gutter={16}>
+          <Col
+            xxl={{ span: 12, offset: 12 }}
+            md={{ span: 18, offset: 6 }}
+            sm={24}
+            xs={24}
+          >
+            <SearchParticipant
+              inputValue={(value) => props.setEnteredFilter(value)}
+            />
+          </Col>
+        </Row>
+        <Row>
+          <Col span={24}>
+            <SortParticipant
+              setParticipantList={(value) => props.setParticipantList(value)}
+              getParticipantList={props.participantList}
+            />
+          </Col>
+        </Row>
+        <br />
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <AccentColorButton
+            accentColor={!isSelecting ? '#1890FF' : '#055202'}
+            disabled={selectedParticipants.length === 0 && isSelecting}
+            label={!isSelecting ? 'Teilnahmezertifikate ausstellen…' : 'Weiter'}
```

Comment:
Are the three dots here on purpose? I think that makes the button look "different" than all the others ...

Reply:
Yeah, I used them on purpose. They usually indicate that there is some form of dialog or confirmation before the action is executed, and are often used in desktop applications; for example the entry "Print..." in some apps' menu bar opens a dialog first. I used them so that people aren't afraid that the button sends *everyone* a certificate, as the user still needs to select the users first.

But, tbh, I don't think many users will realize this, so I might as well just leave it out for visual purposes :D